libraries/resource_providers/api200/connection_template_provider.rb

```diff
@@ -22,6 +22,9 @@ def load_template_from_resource(resource_type, resource_name)
       end
 
       def load_connection_templates
+        resources_set = [@context.associated_ethernet_network, @context.associated_fcoe_network, @context.associated_fc_network,
```

Comment:
Verifying on line 25-26 if only one associated resource has been passed in, if not raising an error. 
This is supposed to prevent the unexpected behavior of just the last associated resource specified being updated instead of all.